#419 Fixing null pointer exception when Remote ATD jar is not started in Remote Host

VinayakaMayura opened the pull request:
Added Remote getDevices & getSimulator inside catch block to avoid NullPointer and ConnectException.

saikrishna321 commented:
@VinayakaMayura can look at the fail fast approach. Just ping to the remote IP and no response just throw Exception() and also fix the CI

VinayakaMayura commented:
1. Moved the exception catch to getRemoteWDHubIP instead of getDevices and removed the multiple catch blocks
2. Removed imports to reduce code complexity, though not seeing any errors in https://app.codacy.com/app/saikrishna321/AppiumTestDistribution/pullRequest?prid=2699142
